app/pulp/app/models/progress.py

```diff
@@ -54,6 +56,7 @@ class ProgressReport(Model):
     done = models.IntegerField(default=0)
 
     task = models.ForeignKey("Task", on_delete=models.CASCADE)
+    suffix = models.TextField(default=None)
```

Comment:
What about a `null=True` instead of `default=None`? As its written now the DB will store `None` in the db unless the plugin writer sets the `suffix`.